kmod/pfckmod.c

```diff
@@ -100,19 +114,43 @@ static const struct bin_attribute   PFC_ATTR_counts     = {
 	.read    = pfcCntRd,
 	.write   = pfcCntWr
 };
-static const struct bin_attribute*  PFC_ATTR_GRP_LIST[] = {
+static const struct bin_attribute*  PFC_BIN_ATTR_GRP_LIST[] = {
```

Comment:
Changed to BIN_ATTR group, contrasting with the new STR_ATTR group which has the verbose and cr4.pce flags.